### PR TITLE
Rename the module to anemic, address the import issue, refactor DCAN

### PR DESCRIPTION
**Pull request by dalgu90**
Hi team. I changed the module name and the related import statements. This PR also addresses the problem of doing `import *` for the Config, and reactors DCAN for the.
- Changed `src` -> `anemic` and changed all the statements related.
- Added the import statements to all the `__init__.py`. Removed the import statements written just for ConfigMapper.
- Make the DCAN model self-contained and remove the `modules.layers` module since it's only used in DCAN.
    - #41 also does refactoring DCAN, but this PR tries to create a directory per model. So let me do it together.

By the second change, if any of the `anemic` submodules are imported, then all the classes of `anemic` will be loaded recursively by the `__init__.py` files so that the ConfigMapper will get the classes.  
FYI, all the import statements are absolute imports, not relative, following [this guide](https://google.github.io/styleguide/pyguide.html#22-imports).

### CODE REVIEW

File: anemic/models/dcan.py [hunk 1]
```diff
@@ -0,0 +1,507 @@
...
+        self.final = nn.Linear(input_size, num_classes)
+        xavier_uniform_(self.final.weight)
+
+    def forward(self, x):
+        m = self.label_wise_attn(x)
+        logits = self.final.weight.mul(m).sum(dim=2).add(self.final.bias)
+        return logits
+
+
+class WordEmbeddingLayer(nn.Module):
```
Comment (SuhasShanbhogue) [inline anemic/models/dcan.py:154]: We can reuse this class from other models?

Comment (abheesht17) [reply anemic/models/dcan.py:154]: We took a call to follow this design: https://huggingface.co/blog/transformers-design-philosophy. So, every file will have all layers which are used in that model.

**Review by abheesht17 — APPROVED** [review]
LGTM! Thanks, @dalgu90, for working on this. Approved the PR :)

File: anemic/datasets/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from . import base_dataset
```
Comment (abheesht17) [inline anemic/datasets/__init__.py:1]: Should we put the full path here as well? `from anemic.datasets. ...`

**Pull request merged**